frontend/src/pages/destructionlist/review/DestructionListReview.tsx

```diff
@@ -186,7 +191,39 @@ export function DestructionListReviewPage() {
     const zaakSelectionSelected = Object.values(zaakSelection).filter(
       (f) => f.selected,
     );
-    setZaakSelectionCount(zaakSelectionSelected.length);
+    setZaakSelection(zaakSelectionSelected.map((f) => f.detail!));
+  };
+
+  const _zaken = {
```

Comment:
I was thinking: is a Tooltip the right component here, or should we re-open the modal with the current value filled in? Also: setting the `placement` to `left ` fixes the issue.